styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blueviolet;
+}
```

Comment:
- [ ] Using element selector in CSS is considered a bad practice. So, I would suggest you use a class selector for styling.